WF-712: Updated repo for new toolbelt changes (#15)
spatialos/FlexibleProjectExample: docs/changelog.md

# Changelog

* **15 Feb 2019:**

>   * Changed the signature of the `spatial alpha cloud upload` and `spatial alpha cloud launch` CLI commands: All positional arguments were replaced by mandatory flags. For more information, please run the spatial CLI with the `--help` flag.

Reviewer: ```suggestion
  * Changed the signature of the `spatial alpha cloud upload` and `spatial alpha cloud launch` CLI commands: replaced all positional arguments with mandatory flags. For more information, please run the spatial CLI with the `--help` flag.
```

What does "run the spatial CLI with the --help flag" mean? Do you just need to run --help from within a SpatialOS project, or do you need to run one of the commands followed by --help?

Author: It's supposed to mean "run the commands with the --help flag, e.g. spatial alpha cloud upload --help"

Reviewer: In that case I'd go for "please run the commands with the `--help` flag" 👍

>   * Removed the `--optimize_for_runtime_v2` CLI flag and added a new `--using_old_loadbalancer` flag for `spatial alpha local launch`. We now optimize for runtime v2 by default. Please set the `--using_old_loadbalancer` flag if you want to start a local deployment with the old runtime.

Reviewer: ```suggestion
  * Removed the `--optimize_for_runtime_v2` CLI flag and added a new `--using_old_loadbalancer` flag for `spatial alpha local launch`. We now optimize for the [new Runtime](https://docs.improbable.io/reference/latest/releases/upgrade-guides/upgrade-runtime) by default. Please set the `--using_old_loadbalancer` flag if you want to start a local deployment with the old Runtime.
```

Author: Done.

>   * Merged the content of the world configuration file (`world.json`) into the [launch configuration](reference/launch-configuration.md). The world configuration is no longer supported going forward.

Reviewer: ```suggestion
  * Merged the content of the world configuration file (`world.json`) into the [launch configuration](reference/launch-configuration.md). We no longer support using the world configuration file.
```

Author: Done.

  * Added a `streaming_query_interval` field to the [launch configuration](reference/launch-configuration.md).
  * Added a `launch_config` field in the [project configuration](reference/project-configuration.md).
  * Added a `loginRateLimit` and `connectionCapacityLimit` field to the [client-worker configuration](reference/client-worker-configuration.md).

>   * Added a `launcherCommand` and `launcherArguments` field to the `uploadConfiguration` section of the [client-worker configuration](reference/client-worker-configuration.md).

Reviewer: ```suggestion
  * Added `launcherCommand` and `launcherArguments` fields to the `uploadConfiguration` section of the [client-worker configuration](reference/client-worker-configuration.md).
```

Author: Done.
